dashboard/app/jobs.go

```diff
@@ -255,7 +254,15 @@ func createBisectJob(c context.Context, managers map[string]bool, reproLevel das
 		if crash == nil {
 			continue
 		}
-		return createBisectJobForBug(c, bug, crash, keys[bi], crashKey)
+		if bug.BisectCause == BisectNot {
+			return createBisectJobForBug(c, bug, crash, keys[bi], crashKey, JobBisectCause)
+		} else if bug.BisectFix == BisectNot {
+			tSinceLast := time.Since(bug.LastTime)
```

Comment:
This should use `timeSince` instead of `time.Since`. `timeSince` supports time mocking in tests. Then tests will be able to control this condition with c.advanceTime.

Reply:
Thanks, fixed